src/repositories/block.repository.ts

```diff
@@ -0,0 +1,6 @@
+import Block from '../models/block.model'
+import { Block as IBlock } from 'interfaces/block.interface'
```

Comment:
thought (non-blocking): still unsure about the naming convention we should follow here. I'd like to make sure to not have a model, interface and implementation etc. named the same. Reading through some ts style guides (also see this detailed answer: https://stackoverflow.com/a/41967120) we should probably stay away from prefixing interfaces with "I"

suggestion: Rename `Block` in `block.model.ts` to `BlockModel`

Reply:
We use I{name} for every interface now (implemented via #22).

Reply:
Then lets stay with this and merge it into here, too